src/Features/Core/Portable/GenerateFromMembers/AbstractGenerateFromMembersService.cs

```diff
@@ -129,7 +129,7 @@ private static bool IsViableProperty(IPropertySymbol property)
 
         private static readonly char[] s_underscore = { '_' };
 
-        protected IMethodSymbol GetDelegatedConstructor(
+        protected virtual IMethodSymbol GetDelegatedConstructor(
             INamedTypeSymbol containingType,
             ImmutableArray<IParameterSymbol> parameters)
```

Comment:
1. who still uses this member?  i don't like it being virtual and overridden in one place.  Can we just have two helpers with appropriate names. i.e. GetMatchingConstructorBasedOnParameterTypes and GetMatchingConstructorBasedOnParameterNames or something like that?  Virtual/override here just ends up being confusing.

Alternatively, this could be abstract/override if this logic is only needed in one other subclass.